res/themes/legacy-dark/css/_legacy-dark.scss

```diff
@@ -83,7 +83,7 @@ $settings-subsection-fg-color: $text-secondary-color;
 $topleftmenu-color: $text-primary-color;
 $roomheader-addroom-bg-color: #3c4556;
 $roomheader-addroom-fg-color: $text-primary-color;
-$icon-button-color: $header-panel-text-primary-color;
+$icon-button-color: $quaternary-content;
```

Comment:
I don't think I see a reason to change to `$quaternary-content`?

Reply:
yikes mb, thx for the catch!